shared/utils/build_remediations.py

```diff
@@ -0,0 +1,429 @@
+#!/usr/bin/python
+
+import subprocess
+import os
+import shutil
+import sys
+
+if sys.version_info < (3,):
+    from sets import Set as set
+
+import argparse
+
+# TODO: environment variables could be passed through parameters
+
+# Configuration
+products_templates = { }
+
+products_templates["shared/templates"] = {
+    "default": (
+        {
+            "csv_dir":  "csv"
+        }, [
+            ("create_kernel_modules_disabled.py", ["bash", "ansible"], "kernel_modules_disabled.csv"),
+            ("create_package_installed.py",       ["bash", "ansible"], "package_installed.csv"),
+            ("create_package_removed.py",         ["bash", "ansible"], "package_removed.csv"),
+            ("create_services_disabled.py",       ["bash", "ansible"], "services_disabled.csv"),
+            ("create_services_enabled.py",        ["bash", "ansible"], "services_enabled.csv"),
+        ]
+    )
+}
```

Comment:
Can all these `product_templates` be autogenerated/autopopulated rather than hard coded?